components/Forms/Select/Select.stories.mdx

```diff
@@ -0,0 +1,121 @@
+import {
+  Meta,
+  ArgsTable,
+  Story,
+  Preview,
+  Canvas,
+  Source,
+} from '@storybook/addon-docs/blocks';
+
+import { Field } from '../Field';
+import { Select, SelectProps } from './Select';
+import {
+  commonProps,
+  options,
+  descriptionOptions,
+  groupedOptions,
+} from './select-story-data';
+
+<Meta title="Components/Forms/Select" component={Select} />
+
+# Select
+
+A single item select input built ontop of [React Select](https://react-select.com/home). Allows for searching, grouping (with labels), and descriptions on options.
+
+### SelectOption
+
+Options are passed into `Select` as an array of `SelectOption` which is defined as:
+
+<Source
+  code={`interface SelectOption {
+  label: string;
+  value: string;
+  meta?: string;
+  options?: SelectOption[];
```

Comment:
Is this intended to be on the `SelectOption` interface?

Reply:
Yup - that's how grouped options work in react-select!

Reply:
Ah, okay! I'm seeing that in the example now, thanks!